src/registrar/views/portfolio_members_json.py

```diff
@@ -0,0 +1,182 @@
+from django.http import JsonResponse
+from django.core.paginator import Paginator
+from django.contrib.auth.decorators import login_required
+from django.db.models import Q
+
+from registrar.models.portfolio_invitation import PortfolioInvitation
+from registrar.models.user import User
+from registrar.models.user_portfolio_permission import UserPortfolioPermission
+from registrar.models.utility.portfolio_helper import UserPortfolioRoleChoices
+
+
+@login_required
+def get_portfolio_members_json(request):
+    """Given the current request,
+    get all members that are associated with the given portfolio"""
+
+    objects = get_member_objects_from_request(request)
```

Comment:
Follow domains and domain_requests json and separate out the id getting into its own method

Reply:
Maybe I'm missing something in the trajectory of this page, but this seems inefficient to me.  Why do we need to separate out the ids?